components/colorslider/stories/template.js

```diff
@@ -0,0 +1,42 @@
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { Template as ColorHandle } from "@spectrum-css/colorhandle/stories/template.js";
+
+
+import '../index.css';
+import '../skin.css';
+
+export const Template = ({
+  rootClass = "spectrum-ColorSlider",
+  customClasses = [],
+  isDisabled = false,
+  isFocused = false,
+  vertical,
+  colorHandleStyle = "--spectrum-picked-color: rgb(255, 0, 0);",
```

Comment:
```suggestion
  colorHandleStyle = {
    '--spectrum-picked-color': 'rgba(255, 0, 0, 0)',
  }
```

Love that this is passed through - can we define it as an object so we can render it through the `styleMap` utility?

Reply:
oh, yes! and I like using style map much more.  
This is passed to ColorHandle so implementation of styleMap it there. I also updated ColorArea to do the same.